README.md

```diff
@@ -1,5 +1,30 @@
 BreathingTasks_PsychoPy
 ==============
-PsychoPy code to display instructions for a 'Breath Hold' and a 'Cued Deep Breathing' task. 
---------------
+PsychoPy code to display visual instructions for a 'Breath Hold' and a 'Cued Deep Breathing' task. 
+
 Written with PsychoPy3.0.5
+
+Tips for using this code:
+--------------
+
+- It is designed to be used alongside recordings of end-tidal CO2 and O2 via a nasal cannula, during MRI, but can be used with other set-ups. 
```

Comment:
```suggestion
- It is designed to be used alongside recordings of end-tidal CO2 and O2 via a nasal cannula, during MRI, but can be used with other set-ups.
- If you are sampling exhaled CO2 and O2, it is good practice to measure the pressure in the laboratory before the session (this will help converting the signal recordings from Volts to mmHg).
```

Reply:
Thanks!